### DISCUSSION TITLE
Include a room type on all new rooms

### DISCUSSION THREAD

**Discussion by q-TheDog** [element-hq/element-meta#579] [Ideas]
**Is your feature request related to a problem? Please describe.**
Now that non-chat clients like MinesTRIX and others are being made, there is a benefit to knowing what each room is made for in order to know which to show on the UI (and where). Currently, rooms made with Element have no type specified like MinesTRIX does for example. This means if a non-chat client wants to know what rooms were made in Element and other clients designed to make chatrooms, it has to just blindly assume all rooms with no type are "chatrooms."

**Describe the solution you'd like**
Include a type in the content section of `m.room.create` events to specify that a room is intended to be a "chatroom." (or whatever term makes sense for Element and the like). Once implemented, suggest developers of other Matrix chat clients to include it as well.

**Describe alternatives you've considered**
Blindly assume all rooms with no type were made in Element and other chat clients.

**Comment by SimonBrandner** [comment]
> it has to just blindly assume all rooms with no type are "chatrooms."

Not necessarily - we could codify that no room type = chat room (https://github.com/matrix-org/matrix-spec-proposals/pull/3827 already sort of this though only for a specific endpoint)

Anyway, this feels more like a https://github.com/matrix-org/matrix-spec issue...
